src/lib/adamant-api/index.js

```diff
@@ -145,32 +213,18 @@ export function getPublicKey(address = '') {
  *   type?: number,
  *   amount?: number
  * }} params
- * @returns {Promise<{success: boolean, transactionId: string}>}
+ * @returns {Promise<{success: boolean, transactionId: string, nodeTimestamp: number}>}
  */
-export function sendMessage(params) {
```

Comment:
We still use `sendSpecialMessage → sendMessage` in `store/index.js`:

https://github.com/Adamant-im/adamant-im/blob/0722d60f071a3043e534b4bd4dde2e522fc26dd8/src/store/index.js#L4-L10